python/lsst/meas/extensions/piff/piffPsfDeterminer.py

```diff
@@ -453,6 +461,13 @@ def determinePsf(
             stars.append(piff.Star(data, None))
 
         if self.config.piffPsfConfigYaml is None:
+            # The following is mostly accommodating unittests that don't have
+            # the filter attribute set on the mock exposure.
+            if hasattr(exposure.filter, "bandLabel"):
+                band = exposure.filter.bandLabel
+            else:
+                band = None
+            spatialOrder = self.config.spatialOrderPerBand.get(band, self.config.spatialOrder)
```

Comment:
I know I suggested this pattern, but a `try`-`except` will allow you to log where it gets this value from. Again, can happen in a later ticket, perhaps when we want to deprecate `spatialOrder`.